src/test/java/org/terasology/launcher/util/TestFileUtils.java

```diff
@@ -80,7 +98,22 @@ public void testNotDirectory() throws IOException {
     @Test
     public void testNoPerms() throws IOException {
         var directory = tempFolder.resolve(DIRECTORY_NAME);
-        Files.createDirectory(directory, PosixFilePermissions.asFileAttribute(Collections.emptySet()));
+        var isPosix = FileSystems.getDefault().supportedFileAttributeViews().contains("posix");
+        Files.createDirectory(directory);
+        var d = directory.toFile();
+
+        // Unfortunately, Win is not POSIX perms compatible, it uses own ACL system
+        List<AclEntry> originalAcl = null;
+        AclFileAttributeView view = null;
+
+        if (isPosix) {
+            assert d.setReadable(false);
+            assert d.setReadable(false);
```

Comment:
duplicated line. was one of those supposed to be `Writable`?

Reply:
Yep, my bad. I'll fix it.